content/_data/changelogs/weekly.yml

```diff
@@ -1812,6 +1812,69 @@
     #        message: >
     #          Developer: Validate project names when creating projects using <code>ItemGroupMixIn#createProjectFromXML()</code>
     #          and <code>ItemGroupMixIn#createProject()</code> method
+- version: "2.102"
+  date: 2018-01-14
+  banner: >
+    JEP-200: &quot;Switch Remoting/XStream blacklist to a whitelist&quot; has been integrated to this release.
+    This change implies a HIGH RISK of regressions in plugins.
+    See <a href="/blog/2018/01/13/jep-200/">this blogpost</a> for upgrade guidelines and the list of affected plugins.
```

Comment:
list of plugins known to be affected, with instructions how to resolve possible problems.